packages/aws-cdk/test/init.test.ts

```diff
@@ -170,7 +169,12 @@ function cliTest(name: string, handler: (dir: string) => void | Promise<any>): v
 }
 
 async function withTempDir(cb: (dir: string) => void | Promise<any>) {
-  const tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), 'aws-cdk-test'));
+  // Intentionally not using any temp dir as base
+  // For some reason, files created in the tmpdir of superchain creates inconsistent
```

Comment:
How is this manifested and why didn't we see this before? Feels like we are solving a problem without fully understanding it...